vllm/v1/executor/multiproc_executor.py

```
@@ -1001,14 +1014,15 @@ def worker_busy_loop(self):
             method, args, kwargs, output_rank = self.rpc_broadcast_mq.dequeue(
                 indefinite=True
             )
+            self.get_method_count += 1
             try:
                 if isinstance(method, str):
                     func = getattr(self.worker, method)
                 elif isinstance(method, bytes):
                     func = partial(cloudpickle.loads(method), self.worker)
-
+                logger.info(f"{self.rank=} exec {method=} {self.get_method_count=}")
```

Ruff (0.16.3)

[warning] Logging statement uses f-string

(G004)